src/Compilers/CSharp/Portable/Symbols/Source/SourceOrdinaryMethodSymbol.cs

```diff
@@ -772,6 +772,11 @@ private DeclarationModifiers MakeModifiers(SyntaxTokenList modifiers, MethodKind
                 allowedModifiers |= DeclarationModifiers.Extern | DeclarationModifiers.Async;
             }
 
+            if (ContainingType.IsStructType())
```

Comment:
>IsStructType [](start = 31, length = 12)

Why is the new modifier limited to members on value types?

Reply:
https://github.com/dotnet/csharplang/issues/1710#issuecomment-453318437

Cc. @jaredpar